src/dialogs/keyboardShortcutsHelp.tsx

```diff
@@ -94,7 +100,9 @@ const KeyboardShortcutsHelp = ({ dialogResult } : { dialogResult: DialogResult<v
 
     const editShortcuts: Array<[React.JSX.Element, string, string]> = [
         [
-            <kbd className="key" key="rename">F2</kbd>,
+            <kbd className="keystroke" key="rename">
+                <kbd className="key">Fn</kbd> + <kbd className="key">F2</kbd>
```

Comment:
I don't fully agree with this, let's not document how laptops can work.

Reply:
I agree with you, dropped